types/crypto-js/index.d.ts

```diff
@@ -30,6 +30,7 @@ declare namespace CryptoJS {
 		salt: string;
 		ciphertext: string;
 		key?: string;
+        toString(encoder?: Encoder): string;
```

Comment:
My apologies.   WebStorm showed it all aligned nicely on my screen.  I'll investigate and learn why the difference in GitHub.  Thanks for accepting it.